tests/integration/__init__.py

```diff
@@ -158,8 +154,16 @@ def remove_cluster():
     global CCM_CLUSTER
     if CCM_CLUSTER:
         log.debug("removing cluster %s", CCM_CLUSTER.name)
-        CCM_CLUSTER.remove()
-        CCM_CLUSTER = None
+        while True:
+            try:
+                CCM_CLUSTER.remove()
+                CCM_CLUSTER = None
+                break
+            except WindowsError:
+                ex_type, ex, tb = sys.exc_info()
+                log.warn("{0}: {1} Backtrace: {2}".format(ex_type.__name__, ex, traceback.extract_tb(tb)))
+                del tb
+                time.sleep(1)
```

Comment:
Specifically, `WindowsError: [Error 5] Access is denied`. This is caused due to the Windows filesystem holding on to files, even though we've released them. Typically the resources are released after a few seconds.